integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -64,23 +64,25 @@ def get_unreviewed_pages_context(self):
         :return: Dictionary containing the context for unreviewed pages
         :rtype: dict
         """
-        unreviewed_pages = (
-            PageTranslation.objects.filter(
-                status=status.REVIEW,
-                language__slug=self.request.region.default_language.slug,
-                page__id__in=Subquery(
-                    self.request.region.non_archived_pages.values("pk")
-                ),
+        if self.request.region.default_language is not None:
+            unreviewed_pages = (
+                PageTranslation.objects.filter(
+                    status=status.REVIEW,
+                    language__slug=self.request.region.default_language.slug,
+                    page__id__in=Subquery(
+                        self.request.region.non_archived_pages.values("pk")
+                    ),
+                )
+                .order_by("page__id", "language__id", "-version")
+                .distinct("page__id", "language__id")
+                .all()
```

Comment:
I think an early return would be cleaner (sorry, I cannot suggest changes for deleted lines, so tthere are some remnants that will have to be deleted manually). (THis goes for all the other functions as well, of course)

```suggestion
         if not self.request.region.default_language:
            return {}

        unreviewed_pages = (
            PageTranslation.objects.filter(
                status=status.REVIEW,
                language__slug=self.request.region.default_language.slug,
                page__id__in=Subquery(
                    self.request.region.non_archived_pages.values("pk")
                ),
            )
            .order_by("page__id", "language__id", "-version")
            .distinct("page__id", "language__id")
            .all()
        )

        return {
            "unreviewed_pages": unreviewed_pages,
            "default_language_slug": self.request.region.default_language.slug,
        }
```